invisible_cities/cities/detsim_simulate_electrons_test.py

```diff
@@ -67,8 +65,13 @@ def test_generate_ionization_electrons(MChits_and_detsim_params):
     assert np.all(secondary_electrons >= 0)
 
     # test is integer
-    isinteger = lambda value: int(value) == value
-    assert all(map(isinteger, secondary_electrons))
+    assert issubclass(secondary_electrons.dtype.type, np.integer)
```

Comment:
This will fail if the dtype is `int`, which is why `np.integer` should be replaced with `(np.integer, int)`.